server/controllers/article.js

```diff
@@ -32,9 +30,28 @@ export const getArticles = async (req, res) => {
     });
 
     if (!articles.length) {
-      handleSuccessResponse(null, 'No article published at the moment', res, 200);
+      handleResponse(null, 'No article published at the moment', res, 'failed', 404);
+    } else {
+      handleResponse(articles, 'Article retrieved successfully', res, 'success', 200);
+    }
+  } catch (error) {
+    const err = new Error('Server error');
+    displayError(err, res, 500);
```

Comment:
you might want to consider returning error instead of `Server error`. 
it makes it easier to debug
```
} catch (error) {
    displayError(error, res, 500);
}
```

Reply:
All other errors have been caught, thus provision for server error is required

Reply:
@Sojisoyoye, i think it would be nice if the error message is been thrown by the server and not just 'server error', you might not know what to troubleshoot.